auslib/web/admin/views/scheduled_changes.py

```diff
@@ -123,7 +121,7 @@ def _delete(self, sc_id, transaction, changed_by):
         if not sc:
             return Response(status=404, response="Scheduled change does not exist")
 
-        form = DbEditableForm(request.args)
+        form = DbEditableForm(connexion.request.args)
```

Comment:
Ah, so this will be removed when you finish converting this endpoint? If so, sounds good for now.